DGX_fair_use_guidelines.md

```diff
@@ -60,3 +60,70 @@ right:
+
+Occasionally, you may need to run a CPU-only job on the server. Keep an eye on the general usage on the cluster and on 
+the node you are using and adjust your limits. It may help to break your processing into separate jobs; that way you can 
+submit as many as the node can handle and wait until they complete before submitting more.
+
+### Memory leaks
+
+In the Run:ai interface, monitor the CPU memory usage of your job. These appear in the "Metrics" charts shown above, 
+below the GPU metrics. If the memory use is climbing monotonically, you 
+have a memory leak. This job risks crashing by running out of memory, being killed by the system, or even bringing down 
+the node it is running on. The `--memory-limit` flag should at least limit the consequences to the job itself. If you 
+see this happening, inspect your code for steps where memory use can accumulate (e.g. objects growing inside a loop). 
+Try clearing those variables once they are no longer needed. Using the garbage collector in Python might help.
+
+## Fair use of disk storage
+
+Storage on the `/nfs` storage device is limited and frequently close to full. Please minimise your use of storage. The 
+following guidelines should help:
+
+* Avoid duplicating files.
+  * If you need to reorganise or rename files for a specific use, consider using links instead of copies.
+* Do not store working / temporary files.
+  * Either delete these or, better still, process files inside your Docker container so they disappear when the job ends.
+* Regularly clean up.
+  * Do you really need `run3_final`, `run3_final_final`, `run3_final_final_final`?
+* Compress when you can.
+  * The `/nfs` storage has compression built in, but explicit compression still saves space.
+  * Archiving a set of thousands of tiny files to a `.tar.gz` can reduce the overhead beyond just total size used.
+* Use a project directory, not your home directory.
+  * This avoids each team member having their own copies of the same files.
+  * If you "get hit by a bus", having your work organised with the intent of sharing it will make it easier for your
+colleagues to pick up where you left off.
+* Consider archiving large datasets.
+  * If you are no longer using a specific dataset, consider moving it off `/nfs`.
+
+## Fair use of disk I/O
+
+The entire cluster runs on a single storage system. If too many processes are trying to read and write from the disk at 
+once then the system will slow down for everyone. Try to optimise your code to avoid repeated reads from the same file. 
+Some modules / data loaders will allow you to explicitly cache data in memory.
+
+This is where we enter slightly controversial territory. Some sysadmins have observed that VScode and possibly Pycharm 
+spawn hundreds of background processes, some of which are constantly checking the filesystem for changes. It has been 
+reported that adding a script to delete `.vscode` processes improved IO performances. Moreover before this change was 
```

Comment:
to this day I don't see any evidence for this claim

Reply:
That's why I said it's controversial. Did you ever discuss it with Eric? He's the one that investigated this rigorously on Jade. If you want to tone down the wording even further, I'm happy to do that, just want the info out there so people can choose.

Reply:
I think its fine